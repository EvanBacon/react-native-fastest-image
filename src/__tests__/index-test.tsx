import React from 'react';
import Image from '..';

import renderer from 'react-test-renderer';

test(`lmao`, () => {
  expect(renderer.create(<Image />).toJSON()).toMatchSnapshot();
});
