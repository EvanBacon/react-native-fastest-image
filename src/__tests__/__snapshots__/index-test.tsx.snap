// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`lmao 1`] = `<Image />`;
